dokka-subprojects/plugin-base/src/test/kotlin/basic/DRITest.kt

```diff
@@ -393,4 +381,99 @@ class DRITest : BaseAbstractTest() {
             )
         }
     }
+
+    @Test
+    fun `#3558 DRI for function with varags parameter`() {
```

Comment:
Based on the discussion above regarding the `toString` of `Vararg`, and what I see, that in some other tests we also validate `DRI.toString`, maybe it makes sense to also add tests for `toString` representation here, in addition to structural comparison, so that we also test `toString` contract?
WDYT?